examples/miscellaneous/miscellaneous_ex12/miscellaneous_ex12.C

```diff
@@ -203,77 +211,81 @@ int main (int argc, char ** argv)
+        // know about the existence of closer non-local nodes.
+        libMesh::MeshBase::const_node_iterator it = mesh.local_nodes_begin();
+        const libMesh::MeshBase::const_node_iterator end = mesh.local_nodes_end();
+        for (; it != end; ++it)
           {
-            nearest_dist_sq = dist_sq;
-            node_C = n;
+            Node *n = *it;
+            const Real dist_sq = (*n-point_C).norm_sq();
+            if (dist_sq < nearest_dist_sq)
+              {
+                nearest_dist_sq = dist_sq;
+                node_C = n;
+              }
           }
-      }
 
-    // Check with other processors to see if any found a closer node
-    unsigned int minrank = 0;
-    system.comm().minloc(nearest_dist_sq, minrank);
-
-    // Broadcast the ID of the closest node, so every processor can
-    // see for certain whether they have it or not.
-    dof_id_type nearest_node_id;
-    if (system.processor_id() == minrank)
-      nearest_node_id = node_C->id();
-    system.comm().broadcast(nearest_node_id, minrank);
-    node_C = mesh.query_node_ptr(nearest_node_id);
-  }
+        // Check with other processors to see if any found a closer node
+        unsigned int minrank = 0;
+        system.comm().minloc(nearest_dist_sq, minrank);
+
+        // Broadcast the ID of the closest node, so every processor can
+        // see for certain whether they have it or not.
+        dof_id_type nearest_node_id;
+        if (system.processor_id() == minrank)
+          nearest_node_id = node_C->id();
+        system.comm().broadcast(nearest_node_id, minrank);
+        node_C = mesh.query_node_ptr(nearest_node_id);
+      }
 
-  // Evaluate the z-displacement "w" at the node nearest C.
-  Number w = 0;
+      // Evaluate the z-displacement "w" at the node nearest C.
+      Number w = 0;
 
-  // If we know about the closest node, and if we also own the DoFs
-  // on that node, then we can evaluate the solution at that node.
-  if (node_C)
-    {
-      const unsigned int w_var = system.variable_number ("w");
-      dof_id_type w_dof = node_C->dof_number (system.number(), w_var, 0);
-      if (w_dof >= system.get_dof_map().first_dof() &&
-          w_dof < system.get_dof_map().end_dof())
-        w = system.current_solution(w_dof);
-    }
-  system.comm().sum(w);
+      // If we know about the closest node, and if we also own the DoFs
+      // on that node, then we can evaluate the solution at that node.
+      if (node_C)
+        {
+          const unsigned int w_var = system.variable_number ("w");
+          dof_id_type w_dof = node_C->dof_number (system.number(), w_var, 0);
+          if (w_dof >= system.get_dof_map().first_dof() &&
+              w_dof < system.get_dof_map().end_dof())
+            w = system.current_solution(w_dof);
+        }
+      system.comm().sum(w);
 
 
-  Number w_C_bar = -E*h*w/q;
-  const Real w_C_bar_analytic = 164.24;
+      Real w_C_bar = -E*h*w/q;
```

Comment:
Nope.  This is just going to get us breaking with --enable-complex again.

When you fix conflicts, it's important to integrate changes, not revert them.  ;-)